topics/reactiveui/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Paul Betts
+display_name: ReactiveUI
+github_url: https://github.com/reactiveui/reactiveui
+logo: reactiveui.png
+released: 2009
+short_description: ReactiveUI is a composable, cross-platform model-view-viewmodel framework.
+topic: reactiveui
+url: https://reactiveui.net
+wikipedia_url: https://en.wikipedia.org/wiki/reactiveui
+---
+ReactiveUI is a composable, cross-platform model-view-viewmodel framework for all .NET platforms that is inspired by functional reactive programming which is a paradigm that allows you to express the idea around a feature in one readable place, abstract mutable state away from your user interfaces and improve improve the testability of your application.
```

Comment:
How about breaking up this sentence? Maybe:

> ReactiveUI is a composable, cross-platform model-view-viewmodel framework for all .NET platforms, inspired by functional reactive programming. It allows you to express the idea around a feature in one readable place, abstract mutable state away from your user interfaces, and improve the testability of your application.

Reply:
Again, you are right. :)
@cheshire137 Thank you for all the comments and for reviewing this. Really appreciate it.
I hope that now it's more fit for the repository, if not I'll fix whatever is needed.